From 15586d6f936ef5355f5b770d5fa494db38232ab2 Mon Sep 17 00:00:00 2001
From: Etienne Carriere <etienne.carriere@foss.st.com>
Date: Wed, 5 Jun 2024 14:35:47 +0200
Subject: [PATCH 0905/1695] core: scmi-server: default value for
 CFG_SCPFW_MOD_STM32_PD

Define a default value for SCP-firmware product specific configuration
switches  CFG_SCPFW_MOD_*.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 66c9d60b5ce6945b23af01f2ab55a34c8051c2dc core: scmi-server: default value for CFG_SCPFW_MOD_STM32_PD]
Change-Id: I25ff3d0b509fc9dd8c001a361c180220ffb49fd7
Signed-off-by: Etienne Carriere <etienne.carriere@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/385439
Domain-Review: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 core/lib/scmi-server/conf-optee-stm32mp1.mk | 3 +++
 core/lib/scmi-server/conf-optee-stm32mp2.mk | 4 ++++
 2 files changed, 7 insertions(+)

diff --git a/core/lib/scmi-server/conf-optee-stm32mp1.mk b/core/lib/scmi-server/conf-optee-stm32mp1.mk
index 9621e5190..6c47e7abf 100644
--- a/core/lib/scmi-server/conf-optee-stm32mp1.mk
+++ b/core/lib/scmi-server/conf-optee-stm32mp1.mk
@@ -24,3 +24,6 @@ CFG_SCPFW_LOG_LEVEL ?= 1
 
 $(call force,CFG_SCPFW_NOTIFICATION,n)
 $(call force,CFG_SCPFW_FAST_CHANNEL,n)
+
+# Default configuration values for product specific modules
+CFG_SCPFW_MOD_PSU_OPTEE_REGULATOR ?= n
diff --git a/core/lib/scmi-server/conf-optee-stm32mp2.mk b/core/lib/scmi-server/conf-optee-stm32mp2.mk
index 471e3f049..ddd9777c1 100644
--- a/core/lib/scmi-server/conf-optee-stm32mp2.mk
+++ b/core/lib/scmi-server/conf-optee-stm32mp2.mk
@@ -32,3 +32,7 @@ CFG_SCPFW_LOG_LEVEL ?= 1
 
 $(call force,CFG_SCPFW_NOTIFICATION,n)
 $(call force,CFG_SCPFW_FAST_CHANNEL,n)
+
+# Default configuration values for product specific modules
+CFG_SCPFW_MOD_PSU_OPTEE_REGULATOR ?= n
+CFG_SCPFW_MOD_STM32_PD ?= n
-- 
2.47.2

